src/ui/QuoteMessageInput/QuoteMessageThumbnail.tsx

```diff
@@ -24,12 +25,51 @@ export default function QuoteMessageThumbnail({ message }: Props): ReactElement
   if (!isFileMessage(message) && !isMultipleFilesMessage(message) || isVoiceMessage(message as FileMessage)) {
     return null;
   }
-  const thumbnailUrl: string = isFileMessage(message)
-    ? (getMessageFirstFileThumbnailUrl(message) || (isImageMessage(message as FileMessage) && getMessageFirstFileUrl(message)))
-    : (getMessageFirstFileThumbnailUrl(message)
-      || isImageFileInfo((message as MultipleFilesMessage).fileInfoList[0])
-      && getMessageFirstFileUrl(message));
-  if ((isThumbnailMessage(message) || isMultipleFilesMessage(message)) && thumbnailUrl) {
+  let thumbnailUrl = getMessageFirstFileThumbnailUrl(message);
+  if (!thumbnailUrl) {
+    if (
+      message.isFileMessage?.()
+      && (
+        isImageMessage(message as FileMessage)
+        || isVideoMessage(message as FileMessage)
+      )
```

Comment:
```suggestion
      message.isFileMessage()
      && (isImageMessage(message) || isVideoMessage(message)
      )
```
Type inference can be done by using message.isFileMessage().

Reply:
So I get isFileMessage is undefined error sometimes when app is running and test also is failing. Idk why but I started avoid using message.isXxxMessage(). Instead I use isXxxMessage(message).

Reply:
In the test or storybook?

Reply:
yes in storybook. Also when running test. Yes we need to find cause and fix it.